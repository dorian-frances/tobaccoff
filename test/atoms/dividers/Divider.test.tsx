import React from 'react';
import { render } from '@testing-library/react-native';
import { Divider } from 'react-native-paper';

describe('<Divider/>', () => {
  it('should render correctly', () => {
    const divider = render(<Divider />);
    expect(divider.toJSON()).toMatchSnapshot();
  });
});
