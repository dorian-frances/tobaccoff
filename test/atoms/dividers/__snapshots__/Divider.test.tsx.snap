// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider/> should render correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "rgba(202, 196, 208, 1)",
        "height": 0.5,
      },
      undefined,
      false,
      false,
      undefined,
    ]
  }
/>
`;
